### DISCUSSION TITLE
textures isnt loaded with models to blender

### DISCUSSION THREAD

**Discussion by BouncyBearVII** [WolvenKit/WolvenKit#1044] [Q&A] [answered]
When i load the models from cyberpunk to blender, texture doesnt add to materials.
My settings 
![image](https://user-images.githubusercontent.com/121569921/209844674-274f1f1f-f86f-4782-89be-d626ce2cbfec.png)
My settings of Blender
![image](https://user-images.githubusercontent.com/121569921/209844869-816cfc2a-a9a5-4a18-8c4b-a62d02ebad15.png)

**Comment by Simarilius-uk** [comment, marked as answer]
Would need to know plugin version, can see you have 8.8 from the wkit pic. the 1.1 plugin needs the nightly wkit.

**Comment by BouncyBearVII** [comment]
![image](https://user-images.githubusercontent.com/121569921/209865525-9af8c493-12c5-4f49-aa84-939015553fdc.png)
That errors is not normal

**Reply by Simarilius-uk** [reply]
Yeah, thats due to using the newest plugin with 8.8, grab a nightly build from https://github.com/WolvenKit/WolvenKit-nightly-releases and redo the export, and it should work.

**Reply by BouncyBearVII** [reply]
and how to export from wolven?

**Reply by Simarilius-uk** [reply]
import/export has moved to the tools menu if thats what you mean.
